cmd/promtool/main.go

```diff
@@ -116,8 +116,27 @@ func checkConfig(t cli.Term, filename string) ([]string, error) {
 				return nil, err
 			}
 		}
-	}
 
+		for _, filesd := range scfg.ServiceDiscoveryConfig.FileSDConfigs {
+			for _, file := range filesd.Files {
+				// try to glob file, since it may be like /path/to/files/*.yml
+				// if glob returns a 0 length collection of matches we can just
+				// check if file exists
+				files, err := filepath.Glob(file)
+				if err != nil {
+					return nil, err
+				}
+				if len(files) != 0 {
+					// there was at least one match for the glob and we can assume checkFileExists
+					// for all matches would pass, we can continue the loop
+					continue
+				}
+				if err := checkFileExists(file); err != nil {
+					t.Infof("  WARNING: error checking existance of file %s for file_sd in scrape job %s: \"%s\"", file, scfg.JobName, err)
```

Comment:
there is no `Warnf` function for `t` (`cli.Term`) but I branched off master and not dev-2.0 which appears to just be using `fmt.Printf`